src/components/CompactServer/CompactServer.tsx

```diff
@@ -29,13 +34,18 @@ export class CompactServer extends React.PureComponent<ICompactServerProps, any>
                 <span className={'server-title'}>
                     <span>{this.props.name}</span>
                 </span>
-                {
-                    this.props.roles.length > 0 &&
-                    <div>
-                        <hr/>
-                        <TagContainer title={''} tags={this.props.roles}/>
-                    </div>
-                }
+                <div>
+                    <hr/>
+                    {
+                        this.props.roles.length > 0 ? 
+                        (
+                            <TagContainer title={''} tags={this.props.roles} />

```

Comment:
wouldn't it be easier to give the tagcontainer, or more specifically the **edit-tags** class a minimum height?